docs/tutorial.rst

```diff
@@ -1109,12 +1112,12 @@ function was used as the objective function.
 
 .. code-block:: shell
 
-    EX_manni_e	-10.0	|manni[C_e]| <=>
-    FRUKIN	10.0	|fru_c[C_c]| + |ATP[C_c]| => |D-Fructose-6-phosphate[C_c]| + |ADP[C_c]| + |H[C_c]|
-    MANNI1PDEH	0.0	|Nicotinamide-adenine-dinucleotide[C_c]| + |manni1p[C_c]| => |D-Fructose-6-phosphate[C_c]| + |H[C_c]| + |Nicotinamide-adenine-dinucleotide-reduced[C_c]|
-    MANNI1PPHOS	10.0	|manni1p[C_c]| + |H2O[C_c]| => |manni[C_c]| + |Phosphate[C_c]|
-    MANNIDEH	10.0	|Nicotinamide-adenine-dinucleotide[C_c]| + |manni[C_c]| => |Nicotinamide-adenine-dinucleotide-reduced[C_c]| + |fru_c[C_c]|
-    MANNIPTS	10.0	|manni[C_e]| + |Phosphoenolpyruvate[C_c]| => |manni1p[C_c]| + |Pyruvate[C_c]|
+    EX_manni_C_e	-10.0	|manni[e]| <=>
```

Comment:
Yes, the `--all-reactions` should be added to that command. It seems like GitHub is changing how comments are done. It doesn't seem to work well right now 😞